### PR TITLE
Improve label overlaps

### LINKED ISSUE #99: chart labels not fully visible
It is not possible to read the labels on yaxis and xaxis, full screen option for larger view of charts would be useful.

chart 1
char 2

### PR DESCRIPTION
**Pull request by kenrota**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

Fixes #99
Relates https://github.com/plotly/plotly.js/issues/1703

## What does this implement/fix? Explain your changes.

* Fix label overlap in Hyperparameter importance graph
  * Before
    before_hyperparameter
  * After
    after_hyperparameter

* Improve label visibility in Parallel coordinate graph, it is not fully visible.
  * Rotate, truncate and break label to avoid overlap
  * I tried hover tooltip but I couldn't it.
    * Relates https://github.com/plotly/plotly.js/issues/1703
  * Before
    before_parcoords 
    * Fictitious long labels for test, base on xgboost sample.
  * After
    after_parcoords

### CODE REVIEW

File: optuna_dashboard/static/components/GraphParallelCoordinate.tsx [hunk 1]
```diff
@@ -96,6 +96,24 @@ const plotCoordinate = (study: StudyDetail, objectiveId: number) => {
       (t.state === "Pruned" && t.values && t.values.length > 0)
   )
 
+  const maxLabelLength = 40
+  const breakLength = maxLabelLength / 2
+  const ellipsis = "…"
```
Comment (kenrota) [inline optuna_dashboard/static/components/GraphParallelCoordinate.tsx:101]: These are my example setting values.

**Review by c-bata — APPROVED** [review]
Perfect :100:

Thank you for your contribution.

**Comment by kenrota** [comment]
Thank you!
